docs(store): Added references to Feature Creators in Reducers and Selectors pages (#3433)
ngrx/platform: projects/ngrx.io/content/guide/store/selectors.md

export const selectFeature = createFeatureSelector&lt;AppState, FeatureState&gt;(fooFeatureKey);
</code-example>

<div class="alert is-important">

> If you're using a [Feature Creator](guide/store/feature-creators), the top level selector is generated automatically (as well as child selectors for each feature state property), so you don't need to use `createFeatureSelector`.

Reviewer: ```suggestion
Using a [Feature Creator](guide/store/feature-creators) generates the top-level selector and child selectors for each feature state property.
```
